src/core/components/models.jsx

```diff
@@ -46,7 +46,7 @@ export default class Models extends Component {
 
     return <section className={ showModels ? "models is-open" : "models"}>
       <h4 onClick={() => layoutActions.show("models", !showModels)}>
-        <span>{isOAS3 ? "Schemas" : "Models" }</span>
+        <span>{isOAS3 ? "schemas" : "models" }</span>
```

Comment:
case change as per design

Reply:
Not a huge fan of this but I'll kick my concerns back up to design instead of holding up this PR